app/locale/sk.coffee

```diff
@@ -50,7 +50,7 @@ module.exports = nativeDescription: "slovenčina", englishDescription: "Slovak",
     have_an_account: "Máš účet?"
     logged_in_as: "Si prihlásený ako"
     computer_science: "Kurz programovania pre každý vek" # {change}
-    ffa: "Zdarma pre všetkých študentov"
+    ffa: "Zdarma pre študentov"
```

Comment:
This title needs to be shorter so it fits in the little green box. After this change it would look like this: 
![screen shot 2018-02-26 at 16 21 39](https://user-images.githubusercontent.com/1964908/36678475-2ef07528-1b11-11e8-80c0-532293bf1092.png)